pkg/sql/backfill.go

```diff
@@ -90,6 +90,14 @@ var columnBackfillBatchSize = settings.RegisterIntSetting(
 	settings.NonNegativeInt, /* validateFn */
 )
 
+// backfillCheckpointInterval is the duration between backfill detail updates.
+var backfillCheckpointInterval = settings.RegisterDurationSetting(
+	"bulkio.backfill.checkpoint_interval",
```

Comment:
should we make it indexbackfill specific or do you think backfill is fine? Just wondering...

Reply:
Renamed.